Auth0/CredentialsManager.swift

```diff
@@ -88,6 +88,32 @@ public struct CredentialsManager {
         return self.storage.deleteEntry(forKey: storeKey)
     }
 
+    /// Revokes the refresh token and, if successful, the credentials are cleared. Otherwise,
+    /// the credentials are not cleared and an error is raised through the callback.
```

Comment:
- Mention the endpoint being called (revoke token)
- Modify the doc. Explain how it will behave when the refresh token is not present. (will clear credentials and not throw errors)

Reply:
@lbalmaceda I've updated this but it's not showing on the diff for some reason:

https://github.com/auth0/Auth0.swift/blob/creds-manager-revoke-token/Auth0/CredentialsManager.swift#L91